kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -509,6 +507,7 @@ export default function useQuizCreation() {
   provide('activeQuestions', activeQuestions);
   provide('selectedActiveQuestions', selectedActiveQuestions);
   provide('allQuestionsSelected', allQuestionsSelected);
+  provide('allSectionsEmpty', allSectionsEmpty);
```

Comment:
I don't think this needs to be provided/injected, as we are only ever consuming it in the place that `useQuizCreation` is invoked.